packages/esm-outpatient-app/src/patient-search/visit-form-queue-fields/visit-form-queue-fields.component.tsx

```diff
@@ -21,139 +20,122 @@ const StartVisitQueueFields: React.FC = () => {
-    }
-  }, [isLoading, allServices]);
-
-  useEffect(() => {
-    setStatus(defaultStatus);
-  }, [defaultStatus]);
-
-  // to sort sortWeight
-  useEffect(() => {
+  const [status, setStatus] = useState(defaultStatus);
+  const [sortWeight, setSortWeight] = useState(() => {
     if (priority === emergencyPriorityConceptUuid) {
-      setSortWeight(1);
-    } else {
-      setSortWeight(0);
+      return 1;
     }
-  }, [priority]);
+    return 0;
+  });
+  const [service, setSelectedService] = useState(() => {
+    if (allServices?.length > 0) {
+      return allServices[0].uuid;
+    }
+    return '';
+  });
 
   return (
-    <div>
+    <div className={styles.container}>
       <section className={styles.section}>
         <div className={styles.sectionTitle}>{t('queueLocation', 'Queue Location')}</div>
-        {isTablet ? (
-          <Layer>
-            <Select
-              labelText={t('selectQueueLocation', 'Select a queue location')}
-              id="queueLocation"
-              name="queueLocation"
-              invalidText="Required"
-              value={selectedQueueLocation}
-              onChange={(event) => setSelectedQueueLocation(event.target.value)}>
-              {queueLocations?.length > 0 &&
-                queueLocations.map((location) => (
-                  <SelectItem key={location.id} text={location.name} value={location.id}>
-                    {location.name}
-                  </SelectItem>
-                ))}
-            </Select>
-          </Layer>
-        ) : (
+        <ResponsiveWrapper isTablet={isTablet}>
           <Select
             labelText={t('selectQueueLocation', 'Select a queue location')}
             id="queueLocation"
             name="queueLocation"
             invalidText="Required"
             value={selectedQueueLocation}
             onChange={(event) => setSelectedQueueLocation(event.target.value)}>
+            {!selectedQueueLocation ? (
+              <SelectItem text={t('selectLocation', 'Select a queue location')} value="" />
+            ) : null}
             {queueLocations?.length > 0 &&
               queueLocations.map((location) => (
                 <SelectItem key={location.id} text={location.name} value={location.id}>
                   {location.name}
                 </SelectItem>
               ))}
           </Select>
-        )}
+        </ResponsiveWrapper>
       </section>
 
       <section className={styles.section}>
         <div className={styles.sectionTitle}>{t('service', 'Service')}</div>
-        {!allServices?.length ? (
-          <InlineNotification
-            className={styles.inlineNotification}
-            kind={'error'}
-            lowContrast
-            subtitle={t('configureServices', 'Please configure services to continue.')}
-            title={t('noServicesConfigured', 'No services configured')}
-          />
-        ) : (
+        <ResponsiveWrapper isTablet={isTablet}>
```

Comment:
Think `ResponsiveWrapper` shouldn't get used here. Didn't seem like this was getting wrapped in a `Layer` beforehand.